core/src/main/java/com/yahoo/oak/EntryHashSet.java

```diff
@@ -70,7 +76,8 @@
     // use union codec to encode key hash integer (first) with its update counter (second)
     private final UnionCodec hashCodec = new UnionCodec(
         KEY_HASH_BITS, // bits# to represent full hash number as integer, bits# to represent update
-        UnionCodec.INVALID_BIT_SIZE); // counter are calculated upon previous parameters (also int)
+        UnionCodec.INVALID_BIT_SIZE, // counter are calculated upon previous parameters (also int
+        Long.SIZE);
```

Comment:
I'm not sure what is the purpose of the `Long.SIZE` here since the entire encoded values should be long so the third item takes the entire space.

Reply:
Long.SIZE means that union size is long, can be int